owa/app/components/__tests__/admissionLocation/rightPanel/admissionLocationForm/addEditBed-test.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import {shallow, mount} from 'enzyme';
+import {mount, shallow} from 'enzyme';
```

Comment:
Just for my personal edification (as I'm not so familiar with React), why is this better?

Reply:
This was not intentional by me. IntelliJ did it automatically when I committed with the auto-organize imports box checked. I figured it would be more work to undo it, and nobody would care either way. (Maybe it's alphabetical?)

Reply:
Sure, it doesn't need to be undone. Thanks for clarifying.